Comanche055/P11.agc

```diff
@@ -858,7 +864,7 @@ SATSTKON	EXTEND
 		RELINT
 		TC	GOPIN		# EXIT THUS BECAUSE WE CAME VIA V46
 
-		SBANK=	PINSUPER	# Added RSB 2009
```

Comment:
This line is actually found nowhere in the original manuscript, but I guess it's required for the program to work correctly, thus I didn't remove it and improve its comment.

Reply:
Ok now, I believe based on @oldmud0's comment on the other PR, I have to remove this line as well.